.travis.yml

```diff
@@ -21,7 +21,11 @@ env:
       ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
       NOT_TEST_BUILD=true
       NOT_TEST_INSTALL=true
-
+    - ROS_DISTRO="indigo"
+      PRERELEASE=true
+matrix:
+  allow_failures:
+    - env: ROS_DISTRO="indigo" PRERELEASE=true
```

Comment:
Interesting, why add these lines?

Reply:
In [this line](https://github.com/ros-industrial/industrial_calibration/pull/112/files#diff-354f30a63fb0907d4ad57269548329e3R26) I'm adding ROS pre-release test feature. It should fail at least for now since this package isn't release ready so that it shouldn't be taken into account for merge decision, but is useful when release becomes an option. 
if release will never be an option this can be omitted, but IMO having it wouldn't hurt except it tends to take a little longer than "normal" jobs so that you may need to wait longer to see the CI result.